[2021-03-25 14:54:16,853] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: EDGAR-Inference-Pipeline.preprocess 2021-03-25T21:54:02.481651+00:00 [queued]>
[2021-03-25 14:54:16,871] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: EDGAR-Inference-Pipeline.preprocess 2021-03-25T21:54:02.481651+00:00 [queued]>
[2021-03-25 14:54:16,871] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-03-25 14:54:16,871] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2021-03-25 14:54:16,872] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-03-25 14:54:16,882] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): preprocess> on 2021-03-25T21:54:02.481651+00:00
[2021-03-25 14:54:16,886] {standard_task_runner.py:52} INFO - Started process 4863 to run task
[2021-03-25 14:54:16,895] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'EDGAR-Inference-Pipeline', 'preprocess', '2021-03-25T21:54:02.481651+00:00', '--job-id', '35', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/edgar_inference.py', '--cfg-path', '/var/folders/_j/s8vbc0cs1fvgclq8hnvtq1wm0000gn/T/tmpcgs4nxpa', '--error-file', '/var/folders/_j/s8vbc0cs1fvgclq8hnvtq1wm0000gn/T/tmpyd8lzgyj']
[2021-03-25 14:54:16,897] {standard_task_runner.py:77} INFO - Job 35: Subtask preprocess
[2021-03-25 14:54:16,947] {logging_mixin.py:104} INFO - Running <TaskInstance: EDGAR-Inference-Pipeline.preprocess 2021-03-25T21:54:02.481651+00:00 [running]> on host prathyushas-mbp.home
[2021-03-25 14:54:16,998] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=EDGAR-Inference-Pipeline
AIRFLOW_CTX_TASK_ID=preprocess
AIRFLOW_CTX_EXECUTION_DATE=2021-03-25T21:54:02.481651+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-03-25T21:54:02.481651+00:00
[2021-03-25 14:54:17,005] {logging_mixin.py:104} INFO - ['CELTF', 'ACFN', 'KGFHF', 'BLFS', 'IRIX', 'BMMJ', 'GHLD', 'TME']
[2021-03-25 14:54:17,435] {credentials.py:1217} INFO - Found credentials in shared credentials file: ~/.aws/credentials
[2021-03-25 14:54:21,971] {python.py:118} INFO - Done. Returned value was: None
[2021-03-25 14:54:21,976] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=EDGAR-Inference-Pipeline, task_id=preprocess, execution_date=20210325T215402, start_date=20210325T215416, end_date=20210325T215421
[2021-03-25 14:54:22,003] {taskinstance.py:1220} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-03-25 14:54:22,057] {local_task_job.py:146} INFO - Task exited with return code 0
